src/plugins/gui/providers/paybisProvider.ts

```diff
@@ -526,8 +528,32 @@ export const paybisProvider: FiatProviderFactory = {
                       console.log(`  tokenId: ${tokenId}`)
                       const nativeAmount = await coreWallet.denominationToNative(amount, displayCurrencyCode)
 
+                      const savedAction: EdgeTxActionFiat = {
+                        type: 'sell',
+                        orderId: invoice,
+                        orderUri: '',
```

Comment:
This prop is optional according to your core branch, should just leave it undefined for clarity

Reply:
Actually good you caught this. There's actually a URI I can put into this.